src/smc_sagews/smc_sagews/julia.py

```diff
@@ -164,15 +164,6 @@ def __reduce__(self):
         """
         return reduce_load_Julia, tuple([])
 
-    def _function_class(self):
```

Comment:
**Scared** Why are you deleting this function?  What the heck?  Isn't it part of what one is supposed to implement for an interface...

Reply:
This was a duplicate method or function, if I remember correctly.

Reply:
Indeed the method is defined twice, once at line 167 and once at line 258:
- https://github.com/sagemathinc/cocalc/blob/master/src/smc_sagews/smc_sagews/julia.py#L167
- https://github.com/sagemathinc/cocalc/blob/master/src/smc_sagews/smc_sagews/julia.py#L258